vspec/vssexporters/vss2id.py

```diff
@@ -68,7 +70,15 @@ def export_node(yaml_dict, node, id_counter, strict_mode: bool) -> Tuple[int, in
     @param strict_mode: strict mode means case sensitivity for static UID generation
     @return: id_counter, id_counter
     """
-    node_id, id_counter = generate_split_id(node, id_counter, strict_mode)
+
+    if not node.constUID:
+        node_id, id_counter = generate_split_id(node, id_counter, strict_mode)
+    else:
+        assert node.constUID.startswith(
+            "0x"
+        ), f"constUID has to begin with '0x': {node.constUID}"
+        assert len(node.constUID) == 10, f"Invalid constUID: {node.constUID}"
```

Comment:
I am considering to move these two `assert` statements to a utlils function and also run them on the generation of UIDs as an additional safety measure..

Reply:
Found a better way to solve this here https://github.com/COVESA/vss-tools/pull/352/commits/f46dd0c2251d69c73a8d14869d528a4db9387196